apps/api/src/app/inbox/usecases/session/session.usecase.ts

```diff
@@ -86,9 +86,12 @@ export class Session {
 
     const token = await this.authService.getSubscriberWidgetToken(subscriber);
 
+    const removeNovuBranding = inAppIntegration.removeNovuBranding || false;
```

Comment:
It will be always `false` unless explicitely specified as `true` on integration entity in db - no migration required

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** shouldn't this be set on the db level? I mean in the schema you can provide the default value, like:
```
removeNovuBranding: {
  type: Schema.Types.Boolean,
  default: false,
},
```
Otherwise, if we use it in different places, we would have to remember to do `inAppIntegration.removeNovuBranding || false`

Reply:
But yeah, this will require migration with setting to false for everyone.

Reply:
@LetItRock That's what I initially did, but then I thought this way it might be a little bit better:

Given the `default: false` value on schema, we would end up in this state:
- `removeNovuBranding` would be on every newly created integration entity (not just on `type: in-app` ones) (the default is automatically set upon creation if im not mistaken)

Without the default value, we are I guess more following the no-schema MongoDB approach, compared to the previous case which is more schema-like:
- we have the `removeNovuBranding` set **only** on `in-app` integration entities, where user interacted with this specific setting, so we are not storing something that not necessary for the other ones

 > Otherwise, if we use it in different places, we would have to remember to do inAppIntegration.removeNovuBranding || false

I think in both cases this is implicit without remembering this check because if you do:
```typescript
if (integration.removeNovuBranding) {
...
}
```
And the property doesn't exist on the entity, it would result in `false`, which follows the idea here that its `true` only if its explicitly set as `true`. 

This line `inAppIntegration.removeNovuBranding || false` is not required, I put it there as a safeguard in case we would somehow manually change the type in database to "string", since Inbox component is quite important part of the app I wanted to make sure its always `false` unless `true`

Reply:
@ChmaraX, thanks for the detailed explanation, I see what you are saying. FYI this flag might be used in the future for the other channels as well, for example, there were plans to introduce some branding to the emails sent with Novu integration. But I think it's ok to proceed with this approach for now.